ui/apps/platform/src/utils/searchUtils.ts

```diff
@@ -222,13 +223,21 @@ export function getListQueryParams(
  * Calculates the API pagination limit and offset parameters given the
  * current page and number of items per page
  */
-export function getPaginationParams(
-    page: number,
-    perPage: number
-): { offset: number; limit: number } {
+export function getPaginationParams({
+    page,
+    perPage,
+    sortOption,
+}: {
+    page: number;
+    perPage: number;
+    sortOption?: ApiSortOption;
+}): Pagination {
+    const sortObject = sortOption ? { sortOption } : {};
```

Comment:
Instead of returning ` { sortObject: undefined }` when the property is not included, we'll omit the property entirely. The BE appears to support both, but removing the property will be cleaner when debugging potential errors.